ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-eus-upgrade-from-4.14.yaml

```diff
@@ -0,0 +1,85 @@
+base_images:
+  assisted-service-scripts:
+    name: ocm-2.9
+    namespace: edge-infrastructure
+    tag: assisted-service-scripts
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: hypershift-operator
+    namespace: hypershift
+    tag: latest
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.14"
+  multi-latest:
```

Comment:
Is `multi-latest` used ?  If not used, we shall remove it, it adds extra workloads to release page

Reply:
yes, use the `multi-latest`

Reply:
Could you point where it's used ?